librz/cons/histogram.c

```diff
@@ -63,6 +64,7 @@ RZ_API RZ_OWN RzStrBuf *rz_histogram_horizontal(RZ_NONNULL RzHistogramOptions *o
 					rz_strbuf_append(buf, " ");
 				}
 			}
+			rz_strbuf_appendf(buf, " %s%3zu", ruler, (255 - threshold));
```

Comment:
One more thing - please make ruler optional, controlled by `RzHistogramOptions` structure.

Reply:
And these options should be created from a new option `scr.hist.ruler`, just like `scr.hist.block` right now.

Reply:
Is it really necessary to make it optional?
It adds extra code, but  cannot think of a situation when it is useful to not have the ruler.

Reply:
Yes, ruler would be unnecessary when the space on the screen is not enough. But it should be enabled by default.

Reply:
i agree. it should be optional

Reply:
Thank you guys for your feedback! I will fix it today